helpers/deploymentConfig.ts

```diff
@@ -0,0 +1,148 @@
+import { contracts as governanceBscMainnet } from "@venusprotocol/governance-contracts/deployments/bscmainnet.json";
+import { contracts as governanceBscTestnet } from "@venusprotocol/governance-contracts/deployments/bsctestnet.json";
+import { ethers } from "hardhat";
+import { DeploymentsExtension } from "hardhat-deploy/types";
+
+import { addresses as venusProtocolBscMainnet } from "../deployments/bscmainnet_addresses.json";
+import { addresses as venusProtocolBscTestnet } from "../deployments/bsctestnet_addresses.json";
+import { convertToUnit } from "./utils";
+
+export enum InterestRateModels {
+  WhitePaper,
+  JumpRate,
+}
+
+export type TokenConfig = {
+  isMock: boolean;
+  name?: string;
+  symbol: string;
+  decimals?: number;
+  tokenAddress: string;
+  faucetInitialLiquidity?: boolean;
+};
+
+export type VTokenConfig = {
+  name: string;
+  symbol: string;
+  asset: string; // This should match a name property from a TokenCofig
+  rateModel: string;
+  baseRatePerYear: string;
+  multiplierPerYear: string;
+  jumpMultiplierPerYear: string;
+  kink_: string;
+  collateralFactor: string;
+  liquidationThreshold: string;
+  reserveFactor: string;
+  initialSupply: string;
+  supplyCap: string;
+  borrowCap: string;
+  vTokenReceiver: string;
+};
+
+export type PreconfiguredAddresses = { [contract: string]: string };
+
+export type DeploymentConfig = {
+  tokensConfig: TokenConfig[];
+  marketsConfig: VTokenConfig[];
+  preconfiguredAddresses: PreconfiguredAddresses;
+};
+
+export type NetworkConfig = {
+  hardhat: DeploymentConfig;
+  bsctestnet: DeploymentConfig;
+  bscmainnet: DeploymentConfig;
+};
+
+const preconfiguredAddresses = {
+  hardhat: {
+    VTreasury: "account:deployer",
+  },
+  bsctestnet: {
+    VTreasury: venusProtocolBscTestnet.VTreasury,
+    NormalTimelock: governanceBscTestnet.NormalTimelock.address,
```

Comment:
Given we are importing the deployments, shouldn't hardhat help us get these addresses by contract name, instead of directly importing the npm packages?

Reply:
It is nice to use ethers to get contracts rather than hardcoding a config because it means this will work with the hardhat network. It will also allow use to test the script by running it locally.

Reply:
VEN-2272